admin/client/dist/js/MemberDatetimeOptionsetField.js

```diff
@@ -32,9 +32,9 @@
 
 		$('.memberdatetimeoptionset .toggle').entwine({
 			onclick: function onclick(e) {
-				jQuery(this).closest('.form__field-description').parent().find('.toggle-content').toggle();
+				jQuery(this).closest('.description').parent().find('.toggle-content').toggle();
```

Comment:
This is just the dist file again so its automatic that it gets overridden. Fix still hasn't been merged properly yet.